streams/src/components/mp4-muxer/isom.ts

```diff
@@ -5,18 +5,29 @@
 // - store(buffer, offset) -> write the value to a buffer
 // - load(buffer, offset) -> read data and store in value
 
-import { MediaTrack } from '../../../utils/protocols/isom'
-
-type BufferMutation = (buffer: Buffer, offset: number) => void
+import { MediaTrack } from '../types/isom'
+import {
+  decode,
+  readUInt8,
+  readUInt16BE,
+  readUInt24BE,
+  readUInt32BE,
+  writeUInt8,
+  writeUInt16BE,
+  writeUInt24BE,
+  writeUInt32BE,
+} from '../utils/bytes'
+
+type BufferMutation = (buffer: Uint8Array, offset: number) => void
 
 // Constants
 const UINT32_RANGE = Math.pow(2, 32)
 
 abstract class BoxElement {
   public byteLength: number
   public value: any
-  abstract copy(buffer: Buffer, offset: number): void
-  abstract load(buffer: Buffer, offset: number): void
+  abstract copy(buffer: Uint8Array, offset: number): void
```

Comment:
Learned something new today. Didn't know about `abstract` classes and methods. :smiley: